lib/lsm_food_reloaded/business_hours/shift.ex

```diff
@@ -0,0 +1,27 @@
+defmodule LsmFoodReloaded.BusinessHours.Shift do
+  use Ecto.Schema
+  import Ecto.Changeset
+
+  schema "business_shifts" do
+    field :start, :time
+    field :end, :time
+    field :modality, Ecto.Enum, values: [:"dine-in", :"take-away", :delivery]
```

Comment:
Shift would be the lowest level here right? Like the "rightmost" entity. Eg: business -> business_hour -> business_shift ?